src/pages/DetailsPage.js

```diff
@@ -0,0 +1,50 @@
+import { useDispatch, useSelector } from 'react-redux';
+import { useParams } from 'react-router-dom';
+import { useEffect } from 'react';
+
+import { getSingleCrypto } from '../redux/currencies/currenciesSlIce';
+import HeroComponent from '../components/HeroComponent';
+import DetailsComponent from '../components/DetailsComponent';
+
+const DetailsPage = () => {
+  const { singleCrypto } = useSelector((state) => state.currencies);
+  const dispatch = useDispatch();
+  const { id } = useParams();
+
+  useEffect(() => {
+    dispatch(getSingleCrypto(id));
+  }, [dispatch, id]);
+  return (
+    <section className="section-container">
+      <HeroComponent info="single crypto info" />
+      <div className="home-intros space">
+        <div className="crypto-img-container">
+          <img
+            src={`https://coinicons-api.vercel.app/api/icon/${singleCrypto?.symbol}`}
+            alt={singleCrypto?.name}
+            className="crpto-img"
+          />
+        </div>
```

Comment:
- [ ] Same here, kindly make sure to follow the provided design, you can use the `opacity` property to add a different color shad for every second column on the Detail page or just add two different shades of the same color to style the information rows

|Your app|Provide design|
|-|-|
|![Screenshot_20230322_163419](https://user-images.githubusercontent.com/25227022/226974244-46b518a1-400b-4ad7-86bb-9291bf787af6.png)|![Screenshot_20230322_163152](https://user-images.githubusercontent.com/25227022/226974133-9993b30a-a0e5-42d9-8af3-5face54e0a80.png)|